index.js

```diff
@@ -0,0 +1,113 @@
+'use strict'
+const path = require('path')
+const statusboard = require('@pkgjs/statusboard')
+
+module.exports = async (c) => {
+  return statusboard(await config(c))
+}
+
+const config = module.exports.config = async (c = {}) => {
+  const buildDir = path.join(__dirname, 'build')
+
+  return Object.assign({
+    // Status Board Config
+    db: path.join(buildDir, 'data', 'data.db'),
+    baseUrl: '/statusboard',
+    outputDirectory: buildDir,
+
+    // Project configs
+    title: 'Express',
+    description: '',
+
+    issueLabels: ['top priority', 'good first issue', 'help wanted', 'discuss', 'meeting'],
+
+    projects: [
+      'expressjs/express',
+      'expressjs/discussions',
+      'expressjs/cors',
+      'expressjs/morgan',
+      'expressjs/response-time',
+      'expressjs/session',
+      'expressjs/multer',
+      'expressjs/body-parser',
+      'expressjs/compression',
+      'expressjs/serve-static',
+      'expressjs/errorhandler',
+      'expressjs/serve-index',
+      'expressjs/csurf',
+      'expressjs/timeout',
+      'expressjs/flash',
+      'expressjs/vhost',
+
+      'pillarjs/router',
+      'pillarjs/cookies',
+      'pillarjs/send',
+      'pillarjs/finalhandler',
+      'pillarjs/path-to-regexp',
+
+      'jshttp/cookie',
+      'jshttp/fresh',
+      'jshttp/content-disposition',
+      'jshttp/mime-db',
+      'jshttp/media-typer',
+      'jshttp/range-parser',
+      'jshttp/type-is',
+      'jshttp/accepts',
+      'jshttp/negotiator',
+      'jshttp/mime-types',
+      'jshttp/compressible',
+      'jshttp/content-type',
+      'jshttp/http-errors',
+      'jshttp/proxy-addr',
+      'jshttp/etag',
+      'jshttp/forwarded',
+      'jshttp/on-finished',
+      'jshttp/on-headers',
+      'jshttp/vary',
+      'jshttp/basic-auth',
+      'jshttp/methods',
+      'jshttp/statuses'
+    ],
+
```

Comment:
```suggestion
    projects: [],
	orgs: [
	"jshttp",
	"pillarjs",
	"expressjs"
	]
```

My preference is to do this by orgs

Reply:
So I dropped some from the list which are unmaintained or deprecated.  We talked about moving them out of the orgs, but haven't yet.  I do agree if we get those cleaned up it would be better to do what you are saying.

Reply:
I did move a couple more out recently (I think 2 months ago) but we do need to go through them. Those repos wouldn't really contribute anything to the stats, though. Is not listing them just to like make the loading process faster?

Reply:
Not listing them was mainly so users or triagers did not spend time on them.  But if we think everything remaining in the repo is good, then the next step would be to add support for specifying orgs and pulling a list of repos to populate data from.  This is not supported today in `@pkgjs/statusboard`, but should be IMO.